librz/core/cmd_analysis.c

```diff
@@ -8625,7 +8625,35 @@ static void cmd_analysis_classes(RzCore *core, const char *input) {
 				break;
 			}
 		}
-		rz_analysis_class_list(core->analysis, input[1]);
+		RzOutputMode mode;
```

Comment:
On one hand, it would be nice to separate this kind of "parsing" into a separate function, on the other hand, it is only done in the "oldshell" code which soon will be completely removed. What do you think, @ret2libc ?

Reply:
There are several places where there is a need to have another switch case like this one. For example, it needs to be done [here](https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_meta.c#L475) as well. So should I define a new function for this in one these files (say in `cmd_analysis.c`) which can then be called everywhere else?